content/en/docs/setup/independent/high-availability.md

```diff
@@ -512,6 +512,22 @@ When this is done, you can follow the [previous step](#kubeadm-init-master0) to
 
 Once kubeadm has provisioned the other masters, you can add them to the load balancer pool.
 
+## Configure kubelet, kube-scheduler and kube-controller-manager to use the virtual IP
+
+After bootstrapping the master nodes using `kubeadm`, the services `kubelet`, `kube-scheduler` and `kube-controller-manager` still connect to local IPs for the API server rather than the virtual IP. This needs to be fixed in the local configuration files on the master nodes. Make sure you replace `<virtual-ip>` with the virtual IP. Patch the configuration files and restart `kubelet` on all master nodes:
```

Comment:
You can pass the VIP into the advertise address to the config and arg override to the components and this step is not necessary.